integ/any_command_combination_test.go

```diff
@@ -108,7 +108,7 @@ func doTestAnyCommandCombinationWorkflow(t *testing.T, backendType service.Backe
 	panicAtHttpError(err, httpResp)
 
 	// skip the timer for S1
-	time.Sleep(time.Second * 2) // wait for a second so that timer is ready to be skipped
+	time.Sleep(time.Second * 5) // wait for a second so that timer is ready to be skipped
```

Comment:
I had to bump this sleep time from 2 to 5 seconds. Otherwise, the test was flaky and sometimes panicked at line 118 due to a 400 error received